src/base/bittorrent/magneturi.cpp

```diff
@@ -76,13 +76,13 @@ MagnetUri::MagnetUri(const QString &source)
     m_hash = m_addTorrentParams.info_hash;
     m_name = QString::fromStdString(m_addTorrentParams.name);
 
-    m_trackers.reserve(m_addTorrentParams.trackers.size());
+    m_trackers.reserve(int(m_addTorrentParams.trackers.size()));
```

Comment:
Anyway we don't use function style cast (at least for non class types).

Reply:
Actually you do in one other place, but I changed it too. And your coding guideline doesn't specify cast style.

Reply:
Still missing in several places.